src/GraphQL.Tests/Shared/JsonStringExtensions.cs

```diff
@@ -19,5 +20,17 @@ public static ExecutionResult ToExecutionResult(this string json, ExecutionError
                 Errors = errors,
                 Executed = executed
             };
+
+        public static Dictionary<string, object> ToDictionary(this string json)
```

Comment:
Nullable annotations.

Reply:
Or just delete method and use `object converted = SystemTextJson.StringExtensions.ToInputs($"{{ \"arg\": {valueString} }}")["arg"];`in AllScalarGraphTypeTests.cs 

?

Reply:
Added NRT anntotations